[2024-11-02T06:26:56.108+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-11-02T06:26:56.134+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_first_dag_v3.first_task scheduled__2024-11-01T00:00:00+00:00 [queued]>
[2024-11-02T06:26:56.154+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_first_dag_v3.first_task scheduled__2024-11-01T00:00:00+00:00 [queued]>
[2024-11-02T06:26:56.155+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 6
[2024-11-02T06:26:56.190+0000] {taskinstance.py:2888} INFO - Executing <Task(BashOperator): first_task> on 2024-11-01 00:00:00+00:00
[2024-11-02T06:26:56.208+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=191) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-11-02T06:26:56.209+0000] {standard_task_runner.py:72} INFO - Started process 192 to run task
[2024-11-02T06:26:56.198+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v3', 'first_task', 'scheduled__2024-11-01T00:00:00+00:00', '--job-id', '117', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmpnvrulna4']
[2024-11-02T06:26:56.214+0000] {standard_task_runner.py:105} INFO - Job 117: Subtask first_task
[2024-11-02T06:26:56.309+0000] {task_command.py:467} INFO - Running <TaskInstance: our_first_dag_v3.first_task scheduled__2024-11-01T00:00:00+00:00 [running]> on host 2d730dc1e71d
[2024-11-02T06:26:56.470+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='johnwell' AIRFLOW_CTX_DAG_ID='our_first_dag_v3' AIRFLOW_CTX_TASK_ID='first_task' AIRFLOW_CTX_EXECUTION_DATE='2024-11-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-11-01T00:00:00+00:00'
[2024-11-02T06:26:56.470+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-11-02T06:26:56.488+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-11-02T06:26:56.488+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo hello world, this is the first task!']
[2024-11-02T06:26:56.495+0000] {subprocess.py:86} INFO - Output:
[2024-11-02T06:26:56.497+0000] {subprocess.py:93} INFO - hello world, this is the first task!
[2024-11-02T06:26:56.497+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-11-02T06:26:56.538+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-11-02T06:26:56.539+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v3, task_id=first_task, run_id=scheduled__2024-11-01T00:00:00+00:00, execution_date=20241101T000000, start_date=20241102T062656, end_date=20241102T062656
[2024-11-02T06:26:56.594+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-11-02T06:26:56.652+0000] {taskinstance.py:3900} INFO - 2 downstream tasks scheduled from follow-on schedule check
[2024-11-02T06:26:56.682+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
